src/Stratis.Bitcoin.Apps.Browser/Dto/StratisApp.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Stratis.Bitcoin.Apps.Browser.Dto
+{
+    public class StratisApp
```

Comment:
Please have a look at my comment in AppsFileService.cs.

Reply:
How does this Blazor project class relate to the comment ?